micromagneticmodel/abstract/container.py

```diff
@@ -400,9 +401,8 @@ def __sub__(self, other):
             if other not in result:
                 msg = f"Term {other.__class__} not in {self.__class__}."
```

Comment:
The message should say that the term other with its attributes (not just the class) is not in the container.

```suggestion
                msg = f"Term {other} not in {self.__class__}."
```